### DISCUSSION TITLE
Import the "Jump to selected object" hotkey to lazer (double-click mousewheel button)

### DISCUSSION THREAD

**Discussion by Skubers** [ppy/osu#24410] [Ideas] [answered]
Basically the title - On stable there's a hotkey to jump the timeline to a selected object if you're not at the object at the moment by double-clicking the mousewheel button.

It's pretty useful

**Comment by bdach** [comment, marked as answer]
This already exists. Double click left mouse on object to seek to it.

**Reply by peppy** [reply]
Also FWIW it was never "middle button" on stable. If that worked it was by chance only.
